distribution/lib/Standard/Base/0.2.32-SNAPSHOT/src/Data/Vector.enso

```diff
@@ -746,16 +755,21 @@ type Vector
              [Pair 1 "a", Pair 2 "b", Pair 1 "c"] . distinct (on = _.first) == [Pair 1 "a", Pair 2 "b"]
     distinct : (Any -> Any) -> Vector Any
     distinct (on = x->x) =
-        builder = here.new_builder
-        existing_elements = HashSet.new
-
-        this.each elem->
-            proj = on elem
-            if existing_elements.contains proj then Nothing else
-                existing_elements.add proj
-                builder.append elem
-
-        builder.to_vector
+        recover = Panic.recover
```

Comment:
```suggestion
        recovered = Panic.recover
```

`recover` is a verb so IMO it suggests a function not a value, I'd suggest changing to a noun.